<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="description" content="Adele Ribeiro's Personal Webpage">
  <meta name="keywords" content="causal inference, causal discovery, causality, CV, data science, AI, healthcare, public health, deep learning, adele, helena, ribeiro">
	<meta name="author" content="Adèle H. Ribeiro">

	<title>Adèle H. Ribeiro</title>

  <!-- Web Fonts -->
  <link href='http://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
	<!-- Bootstrap core CSS -->
	<link href="assets/bootstrap/css/bootstrap.min.css" rel="stylesheet" media="screen">
	<!-- Font Awesome CSS -->
	<link href="assets/css/font-awesome.min.css" rel="stylesheet" media="screen">
	<!-- Animate css -->
  <link href="assets/css/animate.css" rel="stylesheet">
  <!-- Magnific css -->
	<link href="assets/css/magnific-popup.css" rel="stylesheet">
	<!-- Custom styles CSS -->
	<link href="assets/css/style.css" rel="stylesheet" media="screen">
	<!-- Custom styles CSS -->
	<!-- <link href="assets/css/accordion.css" rel="stylesheet" media="screen">  -->
  <!-- Responsive CSS -->
  <link href="assets/css/responsive.css" rel="stylesheet">
  <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
  <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
  <![endif]-->

  <link rel="shortcut icon" href="">
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="assets/images/ico/apple-touch-icon-144-precomposed.png">
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="assets/images/ico/apple-touch-icon-114-precomposed.png">
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="assets/images/ico/apple-touch-icon-72-precomposed.png">
  <link rel="apple-touch-icon-precomposed" href="assets/images/ico/apple-touch-icon-57-precomposed.png">

</head>

<body>

	<!-- Navigation -->
	<header class="header">
		<nav class="navbar navbar-custom" role="navigation">
		      <div class="container-fluid">
		      
		      
		      <!-- BRAND -->
          <div class="navbar-header">
					<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#custom-collapse">
          
          <span class="sr-only">Toggle navigation</span>
          </button>
          <a class="navbar-brand" href="#about"><h1 style="padding-left:15px">Adèle H. Ribeiro</h1></a>
          </div>
		      
		     
				<div class="collapse navbar-collapse" id="custom-collapse">
					<ul class="nav navbar-nav navbar-right">
					  <li><a href="#about">About</a></li>
						<li><a href="#resume">Resume</a></li>
						<li><a href="#fellowships">Fellowships</a></li>
						<li><a href="research.html">Research Group</a></li>
						<li><a href="#conferences">Conferences</a></li>
						<li><a href="#talks">Talks</a></li>
            <li><a href="#media">Media</a></li>
						<li><a href="#tutorials">Lectures</a></li>
            <li><a href="#teaching">Teaching</a></li>
					</ul>
				</div>
				</div>
		</nav>
		<script>
    window.location = "#about";
  </script>
	</header><!-- End Navigation -->
	
	
    <!-- About Section -->
    <section id="about" class="about-section section-padding">
      <div class="container">
        <h2 class="section-title wow fadeInUp">About Me</h2>

        <div class="row">

          <div class="col-md-4 col-md-push-8">
            <div class="biography text-center">
                <div class="myphoto">
                  <img src="assets/images/myphoto.jpg" alt="">
                </div>
              
                <p><h4>Adèle Helena Ribeiro</h4>
                <span>Postdoctoral Researcher</span><br>
                <span>Institute of Medical Informatics</span><br>
		<span>University of Münster</span><br>
                <a class="email" href="mailto:adele.ribeiro@uni-muenster.de">adele.ribeiro@uni-muenster.de</a>
             </div>   
          </div> <!-- col-md-4 -->

          <div class="col-md-8 col-md-pull-4">
            <div class="short-info wow fadeInUp" >
              <h3>About me</h3>
                <p  align="justify">
                I am the head of the <a href="research.html"><strong>CausalAI4Health</strong></a> Research Group at the 
                <a href="https://www.medizin.uni-muenster.de/imi/das-institut.html" target="_blank">
                <strong>Institute of Medical Informatics (IMI), University of Münster</strong></a>, Germany.
                </p>
                <p  align="justify">
                Since October 2022, I have been working with Prof. Dr. Dominik Heider, the current director of IMI. 
                I first joined his research group while still at the Department of Mathematics and Computer Science 
                at Philipps University of Marburg, continuing as a postdoctoral researcher at IMI before
                establishing my own independent research group, supported by the
                <i>Federal Ministry of Research, Technology and Space (BMFTR)</i> of Germany.
                </p>
                <p  align="justify">
                Prior to this, I was a postdoctoral researcher in the Causal Artificial Intelligence Lab at 
                Columbia University, USA, working with Prof. Dr. Elias Bareinboim. Before joining Columbia University,
                I completed a doctoral research internship at the Neuroscience Institute, Princeton University, USA, and worked as
                a postdoctoral researcher at the Laboratory of Genetics and Molecular Cardiology, Heart Institute, University of São Paulo, Brazil.
                 </p>
                <p  align="justify">
                My research focuses on addressing critical challenges in causal and counterfactual inference in real-world domains,
                such as the health sciences, helping bridge the gap between theory and practical applications. 
                This includes developing more robust, scalable, and privacy-preserving causal discovery and effect identification tools,
                with a focus on quantifying uncertainty, integrating background knowledge, and employing human-in-the-loop approaches to
                support personalized healthcare and advance scientific knowledge and decision-making. 
                 </p>

            </div>

          </div>

    </div> <!-- /.row -->
     <h3>Research Interests</h3>
    <div style="padding:30px;">
      <ul class="list-check row" style="list-style: none; padding-left: 0;">
        <li class="col-6 col-md-4">Causal Inference</li>
        <li class="col-6 col-md-4">Explainable AI</li>
        <li class="col-6 col-md-4">Statistical Genetics</li>
        <li class="col-6 col-md-4">Multi-Omics Analysis</li>
        <li class="col-6 col-md-4">Computational Neuroscience</li>
        <li class="col-6 col-md-4">Health and Biomedical Research</li>
      </ul>
    </div>
    
    <div class="download-button text-center" style="margin-left: 20px; margin-top: 10px">
    <a class="btn btn-primary btn-lg" href="assets/pdf/RibeiroAH_CV.pdf" download target="_blank"> <i class="fa fa-download" aria-hidden="true"></i>Curriculum vitae</a>
    </div>

    <BR>
    <div class="social-icons text-center">
      <ul class="list-inline">
      
       <li><a href="https://scholar.google.com/citations?user=Py9MTe4AAAAJ&amp;hl=en" target="_blank">
        <i class="ai ai-google-scholar"></i>
      </a></li>
      
      <li><a href="https://www.medizin.uni-muenster.de/imi/das-institut/team/arbeitsgruppe-machine-learning-for-biomedical-data/dr-rer-nat-adele-ribeiro.html" target="_blank">
        <i class="fa fa-university"></i>
      </a></li>

      <li><a href="https://orcid.org/0000-0002-4523-2062" target="_blank">
        <i class="ai ai-orcid"></i>
      </a></li>                          
        
      <li><a href="http://lattes.cnpq.br/8716519765059982" target="_blank">
        <i class="ai ai-lattes"></i>
      </a></li>
      
      <li><a href="https://github.com/adele" target="_blank">
        <i class="fa fa-github"></i>
      </a></li>
      
      <li><a href="http://linkedin.com/in/adèle-ribeiro-80055817" target="_blank">
        <i class="fa fa-linkedin"></i>
      </a></li>
     <!--  
      <li><a href="https://twitter.com/adelehr" target="_blank">
        <i class="fa fa-twitter"></i>
      </a></li> --> 
      </ul>
    </div> <!-- /.social-icons --> 
    
              
    </div> <!-- /.container -->
  </section><!-- End About Section -->


    <!-- Resume Section -->
    <section id="resume" class="resume-section section-padding">
        <div class="container">
            <h2 class="section-title wow fadeInUp">Resume</h2>
            <div class="row">
                <div class="col-md-12">
                    <div class="resume-title">
                        <h3>Education and Professional Preparation</h3>
                    </div>
                    <div class="resume">
                        <ul class="timeline">
                          <li>
                            <div class="posted-date">
                                <span class="month">Oct 2024</span>
                            </div><!-- /posted-date -->

                            <div class="timeline-panel wow fadeInUp">
                                <div class="timeline-content">
                                    <div class="timeline-heading">
                                        <h3>Postdoctoral Scholar<br></h3>
                                        <span> Institute of Medical Informatics<br>
                                        University of Münster<br>
                                        Münster, Germany </span> 
                                        <div class="timeline-body"> 
                                        <Strong>Project:</Strong> Causal Data Science and Machine Learning in Biomedicine.<br>
                                        <Strong>Advisor:</Strong> Prof. Dominik Heider
                                         </div><!-- /timeline-body -->
                                         
                                         </div><!-- /timeline-heading -->
                                </div> <!-- /timeline-content -->
                            </div><!-- /timeline-panel -->
                        </li>

                            <li class="timeline-inverted">
                                <div class="posted-date">
                                    <span class="month">Oct 2022</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Postdoctoral Scholar</h3>
                                            <span>Laboratory of Data Science in Biomedicine<br>
                                            Philipps-Universit&auml;t Marburg<br>
                                            Marburg, Hesse, Germany </span>
                                            <div class="timeline-body">
                                            <Strong>Project:</Strong> Causal Data Science and Machine Learning in Biomedicine.<br>
                                            <Strong>Advisor:</Strong> Prof. Dominik Heider
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            
                            <li>
                                <div class="posted-date">
                                    <span class="month">Sep 2019</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Postdoctoral Scholar<br></h3>
                                            <span> Causal Artificial Intelligence Laboratory<br>
                                            Data Science / Computer Science Institutes <br>
                                            Columbia University<br>
                                            New York, NY, USA </span> 
                                            
                                            <div class="timeline-body"> 
                                            <Strong>Project:</Strong> Causal Inference in the Health Sciences: from Biased and Heterogeneous Data Collections to Personalized and Improved Patient Outcomes. <br>
                                            <Strong>Advisor:</Strong> Prof. Elias Bareinbom
                                             </div><!-- /timeline-body -->
                                             
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            
                            <li class="timeline-inverted">
                                <div class="posted-date">
                                    <span class="month">Feb 2019</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Postdoctoral Scholar</h3>
                                            <span>Laboratory of Genetics and Molecular Cardiology<br>
                                            Heart Institute (InCor) <br>
                                            University of Sao Paulo<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Project:</Strong> Deep Learning for 12-lead ECG Classification.<br>
                                            <Strong>Advisor:</Strong> Prof. José Eduardo Krieger
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                          
                          
                               <li>
                                <div class="posted-date">
                                    <span class="month">Nov 2018</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Doctor of Philosophy in<br>Computer Science</h3>
                                            <span>Institute of Mathematics and Statistics<br>
                                            University of Sao Paulo (IME-USP)<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            
                                            <Strong>PhD's dissertation:</Strong> 
                                            <a class="timeline-link" href="https://doi.org/10.11606/T.45.2019.tde-15032019-190109" target="_blank">Identification of Causality in Genetics and Neuroscience</a><br>
                                            <Strong>Advisor:</Strong> Prof. André Fujita<br>
                                            <Strong>Co-Advisor:</Strong> Prof. Júlia Maria Pavan Soler<br>
                                            
                                             
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            
                            <li class="timeline-inverted">
                                <div class="posted-date">
                                    <span class="month">Fall 2017</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Doctoral Research Internship</h3>
                                            <span>Neuroscience Institute<br>
                                            Princeton University<br>
                                            Princeton, NJ, USA </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Project:</Strong> Deep learning-based pose representation and dynamics modeling of marmoset monkeys.<br>
                                            <Strong>Advisor:</Strong> Prof. Asif A. Ghazanfar
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            <li>
                                <div class="posted-date">
                                    <span class="month">Jun 2014</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Master of Science in
                                            <br>Computer Science</h3>
                                            <span>Institute of Mathematics and Statistics<br>
                                            University of Sao Paulo (IME-USP)<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Master's thesis:</Strong> 
                                            <a class="timeline-link" href="https://doi.org/10.11606/D.45.2014.tde-04082014-163616" target="_blank">Gene expression analysis taking into account measurement errors and application to real data.</a><br>
                                            <Strong>Advisor:</Strong> Prof. Roberto Hirata Jr.<br>
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            
                            <li class="timeline-inverted">
                                <div class="posted-date">
                                    <span class="month">Dec 2011</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Bachelor of Science in 
                                            Computational<br>and Applied Mathematics</h3>
                                            <span>Institute of Mathematics and Statistics<br>
                                            University of Sao Paulo (IME-USP)<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Senior thesis:</Strong> Analysis of Pyroelectric Infrared (PIR) sensor output signals.<br>
                                            <Strong>Advisor:</Strong> Prof. Roberto Hirata Jr.
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>

                      </ul>
                    </div>
                </div>
            </div><!-- /row -->
        </div><!-- /.container -->
    </section><!-- End Resume Section -->



    <!-- Fellowships Section -->
    <section id="fellowships" class="section section-padding">
      
      <div class="container">
      <h2 class="section-title wow fadeInUp">Awards and Fellowships</h2>

    <div class="list-info wow fadeInUp">
      
      <table class="table-borderless" style="width:100%">
        
          <tr>
            <td style="vertical-align:top"> Sep 2021 </td>
            <td>  <a target="_blank" href="https://www.daad.de/en/the-daad/postdocnet/fellows/fellows/">DAAD Postdoc-NeT-AI Fellowship</a><br>
            DAAD Artificial Intelligence Networking (AInet) Fellowship<br>
            Federal Ministry of Education and Research, Germany<br>
            </td>
          </tr> 

           <tr>
            <td style="vertical-align:top"> Sep 2019 - Aug 2022</td>
            <td> Postdoctoral Research Fellowship <br>
             Causal Artificial Intelligence Lab <br>
             Department of Computer Science &amp; Data Science Institute, Columbia University, New York, NY, USA<br>
            </td>
          </tr> 
          		
           <tr>
            <td style="vertical-align:top"> Jan 2019 - Aug 2019 </td>
            <td> Postdoctoral Research Fellowship<br>
             <!-- Laboratory of Genetics and Molecular Cardiology<br> -->
             <!-- Heart Institute (InCor), University of Sao Paulo (USP), Sao Paulo, SP, Brazil <br>-->
             Coordination for the Improvement of Higher Education Personnel (CAPES), Brazil
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Sep 2017 - Dec 2017 </td>
            <td> PhD Visiting Student at Princeton University<br>
            <!-- Neuroscience Institute, Princeton University, NJ, USA <br>-->
            Coordination for the Improvement of Higher Education Personnel (CAPES), Brazil
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Aug 2014 - Jul 2018 </td>
            <td> PhD Graduate Research Scholarship<br>
             <!-- Department of Computer Science, Institute of Mathematics and Statistics, University of Sao Paulo (USP), Sao Paulo, SP, Brazil <br>-->
             Coordination for the Improvement of Higher Education Personnel (CAPES), Brazil
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Mar 2012 - Feb 2014 </td>
            <td> MSc Graduate Research Scholarship<br>
             <!-- Department of Computer Science, Institute of Mathematics and Statistics, University of Sao Paulo (USP), Sao Paulo, SP, Brazil <br> -->
             Coordination for the Improvement of Higher Education Personnel (CAPES), Brazil
            </td>
          </tr> 
          
        </table>
    </div>
 
          
      </div><!-- /.container -->
   </section><!-- End Fellowships Section -->

  
  
    <!-- Conferences Section -->
    <section id="conferences" class="section section-padding">
      
      <div class="container">
      <h2 class="section-title wow fadeInUp">Participation in Conferences</h2>

      
      <div class="panel-group" id="accordionConf" role="tablist" aria-multiselectable="true">

        <div class="panel panel-default">
          <table style="width:100%">
            <tr>
              <td class="panel-heading" role="tab" id="headingConf8">
                da Silva, T., Silva, E., Góis, A., Heider, D., Kaski, S. and Mesquita, D.*, <Strong>Ribeiro, A. H.</Strong> (2024). 
                Human-Aided Discovery of Ancestral Graphs.<i> LXAI Workshop at NeurIPS 2024 </i>, Vancouver Convention Centre, Vancouver, British Columbia, Canada. 
                 <a href="assets/pdf/posterLXAIatNeurIPS2024.pdf" download target="_blank">(Poster Presentation)</a> </td>
              <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf8" aria-expanded="false" aria-controls="collapseConf8"></a></td>
            </tr> 
          </table>
          <div id="collapseConf8" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf8">
              <div class="panel-body">      
                We introduce Ancestral GFlowNets (AGFNs) as a new amortized inference method for sampling from a belief distribution on the space of ancestral graphs. 
                We develop the first human-in-the-loop framework for ancestral causal discovery (CD). 
                We design an optimal strategy for elicitation of an expert's feedback regarding the nature of a specific causal relationship among the observed variables,
                We demonstrate that our human-aided CD method drastically outperforms traditional CD algorithms after just a few expert interactions.    
              </div>
            </div>
        </div>

        <div class="panel panel-default">
          <table style="width:100%">
            <tr>
              <td class="panel-heading" role="tab" id="headingConf7">
                <Strong>Ribeiro, A. H.</Strong>, Fehse, L., Winter, N., Welzel, M., Kircher, T., Thanarajah, S. E., Dannlowski, U., Heider, D., Hahn, T. (2024).
                 Uncovering Gut Microbiota's Causal Role in Major Depressive Disorder. <i> 13th Sino-German Frontiers of Science Symposium (SINOGFOS)</i>, Shanghai, China. Supported by the Chinese Academy of Sciences and Humboldt Foundation.
                 <a href="assets/pdf/posterSINOGFOS2024.pdf" download target="_blank">(Poster Presentation)</a> </td>
              <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf7" aria-expanded="false" aria-controls="collapseConf7"></a></td>
            </tr> 
          </table>
          <div id="collapseConf7" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf7">
              <div class="panel-body">     
                Major Depressive Disorder (MDD) is a multifaceted mental health condition. Despite numerous studies highlighting a significant association between MDD and the gut microbiome, 
                it remains unclear whether these associations play a causal role in MDD development. In this study, we conducted a differential abundance
                analysis (DAA) followed by a causal analysis of the DFG FOR2107 dataset (https://for2107.de/), which includes 1,269 patients.   
                We highlight two important contributions: (1) Through a meticulous application of the FCI algorithm, we identified that Eggerthella and Hungatella causally contribute to MDD, while Coprobacilius indirectly causes MDD via Eggerthella.
                (2) Obesity not only affects MDD but also confounds between taxa variables and MDD. Using effect identification tools, we show the interventional probability of MDD increases with the abundance of Eggerthella and Hungatella.  
              </div>
            </div>
        </div>

      <div class="panel-group" id="accordionConf" role="tablist" aria-multiselectable="true">

        <div class="panel panel-default">
          <table style="width:100%">
            <tr>
              <td class="panel-heading" role="tab" id="headingConf6"> Levshina, N., <Strong>Ribeiro, A. H.</Strong> (2023). Who did What to Whom: Measuring and explaining cross-linguistic differences. 
                <i>10th International Contrastive Linguistics Conference</i> <a href="https://easychair.org/smart-program/ICLC-10/2023-07-19.html" download target="_blank">(ICLC-10)</a> 
                 Mannheim, Germany. July, 2023.
                 <a href="assets/pdf/abstractICLC10.pdf" download target="_blank">(Conference Abstract)</a> </td>
              <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf6" aria-expanded="false" aria-controls="collapseConf6"></a></td>
            </tr> 
          </table>
          <div id="collapseConf6" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf6">
              <div class="panel-body">             
            </div>
            </div>
        </div>
       

        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="headingConf5"><Strong>Ribeiro, A. H.</Strong>, Sato, J. R., Fujita, A. (2018). Granger Causality Between Graphs and Applications in Functional Brain Networks.  <i> X-Meeting - 14th International Conference of the AB3C <i>,  October 24th - 26th, 2018, São Pedro, SP, Brazil. <a href="assets/pdf/posterXmeeting2018.pdf" download target="_blank">(Poster Presentation)</a> - Best Poster Award  </td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf5" aria-expanded="false" aria-controls="collapseConf5"></a></td>
          </tr> 
        </table>
        <div id="collapseConf5" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf5">
            <div class="panel-body">
                Networks are everywhere, from social to biological sciences. Usually these networks are represented by graphs, i.e., mathematical objects composed of a set of vertices and a set of edges. However, a vast number of natural networks are dynamic and current methods typically ignore a third key component: time. This fact requires statistical approaches to analyze them appropriately. 
<br><br>
In this context, we propose a methodology to identify Granger causality among graphs. By assuming that graphs are generated by models whose parameters are random variables, we define that a time series of graphs y_{i,t} does not Granger cause another time series of graphs y_{j,t} if the parameters of the model for y_{i,t} does not Granger cause the parameters of the model for y_{j,t}. The problem is that the models that generate the graphs are usually unknown and consequently the parameters cannot be estimated. However, for some random graph models, such as Erdös-Rényi, geometric, regular, Watts-Strogatz, and Barabási-Albert, it is known that the spectral radius (the largest eigenvalue of the adjacency matrix of the graph) is a function of the model parameters. For example, for the Erdos-Renyi random graph model, which is defined by the parameters n, number of vertices, and p, probability of two random vertices are connected, the spectral radius is known to be np. 
<br><br>
Based on this idea, we propose to identify Granger causality between time series of graphs by fitting a vector autoregressive model (VAR) to the time series of spectral radii. By an extensive simulation study, we show that the methodology has good accuracy, particularly for large graphs and long time series. In addition, we show that the spectral radius performed better than other centrality measures, such as, degree, eigenvector, betweenness, and closeness centralities. Finally, we applied the methodology to identify Granger causality between brain networks.
            </div>
          </div>
        </div>
        
        
        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="headingConf3"><Strong>Ribeiro, A. H.</Strong>, Soler, J. M. P., Fujita, A. (2018). Learning Genetic and Environmental Causal Graphical Models in Family-Based Studies. <i> Abstracts for the XIXth International Biometric Conference </i>, July 8-13, 2018, Barcelona, Spain, International Biometric Society. <a href="https://higherlogicdownload.s3.amazonaws.com/BIOMETRICSOCIETY/713ac962-588b-42d5-940f-47ae32f0b28c/UploadedImages/USB_Abstracts.pdf" download target="_blank">Conference Abstract.</a> (Contributed Talk)</td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf4" aria-expanded="false" aria-controls="collapseConf4"></a></td>
          </tr> 
        </table>
        <div id="collapseConf4" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf4">
            <div class="panel-body">
              To unravel the biological mechanism underlying complex traits and diseases, it is crucial to understand how the related phenotypes are associated with each other and how they are influenced by genetic and environmental factors. Probabilistic graphical models (PGMs) are widely used to describe relationships among variables (phenotypes) in a
very intuitive and mathematically rigorous way. On the other hand, family-based studies are usually conducted to assess the influence of genetic and environmental factors on phenotypes. In this case, the polygenic model can be used to decompose the phenotypic variability into two variance components: one polygenic, for capturing the
variability across families, and one environmental, for capturing the residual variability.
Some algorithms for learning PGMs from observational data, known as structure learning algorithms, are strongly based on a conditional independence test. Considering the case where the observations are independent and pnormally distributed, the null hypothesis of conditional independence can be tested using classical tests for zero partial correlation and PGMs can be learned under Markov-properties equivalence. However, in family-based studies, measurements of related individuals are correlated and such dependence structure must be taken into account to obtain appropriate test statistics.
<br><br>
Based on the Gaussian univariate polygenic model, we derived an estimator for the partial correlation coefficient taking into account the family dependence structure and present a decomposition of the partial correlation coefficient according to the contribution of the genetic and environmental effects. Also, we derived zero partial correlation tests for these coefficients and extended the Meinshausen and Buhlmann (2006)'s approach, which learns undirected PGMs from Vertex Neighborhoods, and the IC (Pearl, 2000) / PC (Spirtes et al., 2000) algorithm, which learns directed PGMs, for learning genetic and environmental PGMs from observational family data. The performance of the proposed methodologies was assessed by using 100 replicates of simulated data, based on the
Framingham Heart Study, provided by the Genetic Analysis Workshop (GAW) 13 in problem 2.
            </div>
          </div>
        </div>

 
        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="headingConf3">Soler, J. M. P., <Strong>Ribeiro, A. H.</Strong>, Jahnke, M. R. (2017). A produção da cerveja produzindo conhecimento. <i> 3o Congresso de Graduação da USP</i>, São Paulo, SP, Brazil. </i> <a href="assets/pdf/posterProdCerv2017.pdf" download target="_blank">(Poster Presentation)</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf3" aria-expanded="false" aria-controls="collapseConf3"></a></td>
          </tr> 
        </table>
        <div id="collapseConf3" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf3">
            <div class="panel-body">
              A cerveja é parte da história da humanidade e remonta dos legados deixados pelos antigos sumérios, egípcios, mesopotâmios e ibéricos há, pelo menos, 6000 a.C. Apesar
disso, longe de ser considerado um processo estável, a produção da cerveja evolui e
aprimora-se constantemente, a ponto de, atualmente, motivar uma indústria artesanal
em franca expansão que, devido às inúmeras fontes de variabilidade intrínsecas,
potencializa o espírito curioso e criativo do alquimista e o refinamento sensorial de
indivíduos, independentemente de idade, gênero, condição social, etc.
<br><br>
Identificamos nesse universo uma janela ampla para o despertar do entusiasmo ao
aprendizado de alunos do 3o ano da Graduação em Estatística na disciplina de
Planejamento de Experimentos (MAE 0317) que abraçaram, imediata e vigorosamente,
a proposta de produzirem cerveja como veículo ilustrativo transversal dos conceitos e
ferramentas abordados na disciplina. Assim, formalizamos um projeto conjunto, a ser
planejado e executado durante o 1º semestre de 2017, em sala de aula e em campo,
envolvendo o professor, alunos, a monitoria e especialistas na produção de cerveja.
<br><br>
A ideia é combinar estatisticamente respostas que mensuram a qualidade da cerveja,
tais como, densidade, estabilidade da espuma e experiência sensorial (corpo, amargor,
doçura, aroma, transparência, etc.) contra fatores de variabilidade que podem ser
controlados experimentalmente, tais como, a temperatura de cozimento, a
carbonatação e a maturação. Considerando os resultados preliminares obtidos até agora e as perspectivas manifestas, acreditamos que o projeto permite trabalhar a percepção do conteúdo da disciplina pelo aluno, de tal forma a transformar o aprendizado de conceitos teóricos densos em uma experiência prazerosa, estimulante e interativa.
            </div>
          </div>
        </div>

        
        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="headingConf3"><Strong>Ribeiro, A. H.</Strong>, Soler, J. M. P., Fujita, A. (2016) A Comparative Study of Algorithms for Learning Causal Genotype–Phenotype Networks. <i> Abstracts for the XXVIIIth International Biometric Conference </i>, July 10-15, 2016, Victoria, British Columbia, Canada, International Biometric Society. ISBN 978-0-9821919-4-1. <a href="assets/pdf/posterIBC2016.pdf" download target="_blank">(Poster Presentation)</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf3" aria-expanded="false" aria-controls="collapseConf3"></a></td>
          </tr> 
        </table>
        <div id="collapseConf3" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf3">
            <div class="panel-body">
              A challenging task in biomedical research is to understand precisely the complex network of causal associations among phenotypes and outcomes.
        Experimental studies such as clinical trials are the most trustworthy method of causality assessment. However, it may be unfeasible to carry out randomized experiments to discover all possible causal relationships when the number of variables is large. In systems genetics, causal inference is supported by Mendelian randomization, which provides a natural randomization process where genotypes, rather than treatments, are randomly allocated to individuals. Furthermore, genetic variants robustly associated with phenotypes can be seen as instrumental variables, allowing inferences on the causal relation between phenotypes and outcomes. 
        <br><br>
        In this work, we made a comparative study among four recent algorithms that use genetic variants as instrumental variables for learning the structure of a genotype-phenotype network, namely, (i) QTL-directed Dependency Graph (QDG), (ii) QTL-driven phenotype network (QTLnet), (iii) Sparsity-aware Maximum Likelihood (SML), and (iv) QTL+Phenotype Supervised Orientation (QPSO). These algorithms are similar in the sense that they use QTL information to determine the causal direction among phenotypes. However, they were designed under different assumptions and therefore some may be more suitable than others for a particular biological application. By simulation studies, we investigated advantages and limitations of these methodologies, under different configurations. Finally, we applied the algorithms to real data involving cardiovascular phenotypes of F2 rats and compared the inferred causal networks.
            </div>
          </div>
        </div>

        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="headingConf2">           
            Swinka, B. B.,  Carvalho, C. M., Weihermann, A, Schuck, D. C., Boldrini, N., Silva, V. V., Costa, M. T., <Strong>Ribeiro, A. H.</Strong>, Fujita, A., Brohem C. A., and Lorencini M. (2015). Analysis of extracellular-matrix and cell-adhesion genes modulated by mechanical massage applied in combination with a cosmetic emulsion. <i> Supplement issue of the Journal of Investigative Dermatology, Epidermal Structure &amp; Barrier Function </i>, v. 135, p. S58-S69, 74th Annual Meeting of the Society for Investigative Dermatology (SID 2015), Atlanta, GA, USA. Conference Abstract: <a href="https://doi.org/10.1038/jid.2015.71" download target="_blank">https://doi.org/10.1038/jid.2015.71</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf2" aria-expanded="false" aria-controls="collapseConf2"></a></td>
          </tr> 
        </table>
        <div id="collapseConf2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf2">
            <div class="panel-body">
                Massage therapies are associated with pathological improvements, and have also been extensively used for esthetic purposes. This study aimed to evaluate part of the molecular mechanisms involved in massage by investigating modulation of gene expression associated with cell adhesion and the ECM (extracellular matrix) induced by esthetic massage combined with a cosmetic emulsion. Thirteen female volunteers clinically characterized as having grade II or III cellulite were recruited and were subjected to skin biopsies in the gluteofemoral region before and after treatment. Each volunteer’s leg was considered an experimental unit to reduce individual variability. The study population was divided into: (1) legs treated with a cosmetic emulsion and (2) legs treated with a cosmetic emulsion and massage. Examination of 84 genes analyzed by qPCR revealed a predominance of up-regulation in individuals treated with the emulsion and massage in comparison to individuals treated only with the emulsion (fold change > 1.5, and p < 0.05). The main genes modulated were: ECM proteases (ADAMTS8, MMP1, MMP3, MMP9 and MMP11), transmembrane molecules (HAS1, ITGAL), adhesion molecules (COL8A1 and LAMA1) and cell-matrix adhesion molecules (ADAMTS13). Concluding, the combination (cosmetic emulsion and massage) is therefore recommended to increase the effectiveness of a product and obtain the desired benefits in the treatment of skin disorders such as cellulite. The lack of scientific data on this technique can very often lead to skepticism among health professionals and even patients or consumers of cosmetic treatments. This study helps to elucidate some of the molecular phenomena associated with this therapy.
            </div>
          </div>
        </div>
    
        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="headingConf1"><Strong>Ribeiro, A. H.</Strong>, Hirata Jr., R., Soler, J. M. P. (2014). Two-color microarray data analysis taking into account probe-level inaccuracies.  <i> ISCB-Latin America X-Meeting on Bioinformatics with BSB and SoiBio <i>, Belo Horizonte, MG, Brazil. <a href="assets/pdf/posterXmeeting2014.pdf" download target="_blank">(Poster Presentation)</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordionConf" data-target="#collapseConf1" aria-expanded="false" aria-controls="collapseConf1"></a></td>
          </tr> 
        </table>
        <div id="collapseConf1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingConf1">
            <div class="panel-body">
                Most analyses of two-color microarray data are based on point estimation of the log-ratio of the two channel intensities. These estimates, commonly named M values, are conventionally obtained from some location measure of the pixel intensities of each channel, ignoring any imprecision. It is well known that the microarray technology is associated with many noise sources, and it has been shown that improved inferences can be obtained by including the inaccuracies involved and propagating them to downstream analysis. Using the multivariate delta method, we propose new estimators for the mean and the variance of the M values that take into account the probe-level inaccuracies in the analysis.
            </div>
          </div>
        </div>
    </div>
    
    </div><!-- /.container -->
   </section><!-- End Conferences Section -->



 
    <!-- Talks Section -->
    <section id="talks" class="section section-padding">
      
      <div class="container">
      <h2 class="section-title wow fadeInUp">Invited Talks</h2>
     
    <div class="list-info wow fadeInUp">
      
      <table class="table-borderless" style="width:100%">

          <tr>
            <td style="vertical-align:top"> Dec 2024 </td>
            <td> L3S Research Center, Leibniz University, and CAIMed <br>
              L3S Research Center, Leibniz University, and Lower Saxony research Center for Artificial Intelligence and Causal Methods in Medicine (CAIMed), Hannover, Germany<br>
              <i> <strong>Ribeiro, A. H.</strong> From Theory to Practice: Advancing Causal Inference for Real-World Applications in Health Sciences. </i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Oct 2024 </td>
            <td> Seminar at Université Grenoble Alpes <br>
              Institut d'Informatique et Mathématiques Appliquées de Grenoble (IMAG), France<br>
              <i> <strong>Ribeiro, A. H.</strong> Recent Advances in Causal Inference under Limited Domain Knowledge. </i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Jun 2024 </td>
            <td> <a target="_blank" href="https://mathcal.ma.tum.de/mc/showtalks?filter=0&id=2304">TUM Seminar on Statistics and Data Science</a> <br>
              Department of Mathematics, Technical University of Munich (TUM), Germany <br>
              <i> <strong>Ribeiro, A. H.</strong> Recent Advances in Causal Inference under Limited Domain Knowledge </i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> May 2024 </td>
            <td> <a target="_blank" href="https://68rbras.com.br/miniconferencias/"> 68th Annual Meeting of RBras</a> <br>
              Brazilian Region of the International Biometrics Society (RBras), ESALQ/USP, in Piracicaba, SP, Brazil <br>
              <i> <strong>Ribeiro, A. H.</strong> From Observations to Causality: Recent Advances and Ongoing Challenges </i>
            </td>
          </tr> 

        <tr>
          <td style="vertical-align:top"> Aug 2023 </td>
          <td> <a target="_blank" href="https://emap.fgv.br/en/event/recent-advances-causal-inference-under-limited-domain-knowledge/"> Seminar at FGV EMAp  </a> <br>
            School of Applied Mathematics of Getulio Vargas Foundation (FGV EMAp), Rio de Janeiro, Brazil. <br>
            <i> <strong>Ribeiro, A. H.</strong> Recent Advances in Causal Inference under Limited Domain Knowledge. </i>
          </td>
        </tr> 

        <tr>
          <td style="vertical-align:top"> Apr 2023 </td>
          <td> <a target="_blank" href="https://crl-tuebingen-2023.github.io/speakers/"> Workshop on Causal Representation Learning </a>  <br>
            Max Planck Institute for Intelligent Systems, Tübingen, Germany <br>
            <i> <strong>Ribeiro, A. H.</strong>. Effect Identification in Cluster Causal Diagrams.</i>
          </td>
        </tr> 

        <tr>
          <td style="vertical-align:top"> Aug 2022 </td>
          <td> DAAD Postdoc-NeT-AI Tour, Germany <br>
            Institute of Information Systems &amp;  Institute for Medical Biometrics and Statistics at the University of Lübeck; <br>
            Institute for Computational Systems Biology at the University of Hamburg;<br>
            Centre for Cognitive Science at TU Darmstadt;<br>
            Center for Systems Biology and Department of Computer Science at TU Dresden; and<br>
            Helmholtz Center Munich <br>
            <i> <strong>Ribeiro, A. H.</strong>. Causal Inference from Observational Data in Partially Understood Domains. </i>
          </td>
        </tr> 

        <tr>
          <td style="vertical-align:top"> Aug 2022 </td>
          <td> <a target="_blank" href="https://twitter.com/Heiderlab/status/1562079688725614592?t=YzRaRbeaDXMn9LFjBFKSfQ&s=08"> Future Bioinformatics Workshop, Germany </a> <br>
            <i> <strong>Ribeiro, A. H.</strong>. Causal AI: Towards Explainable, Generalizable, and Trustworthy Decision-Making. </i>
          </td>
        </tr> 

          <tr>
            <td style="vertical-align:top"> Jun 2022</td>
            <td> Columbia DSI Scholars - Summer Research Bootcamp 2022<br>
              Data Science Institute, Columbia University <br>
              <i> <strong>Ribeiro, A. H.</strong> An Overview on Causal Data Science.</i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> May 2022</td>
            <td>  <a target="_blank" href="https://www.pipges.ufscar.br/seminars/may-27-2022-causal-effect-identification-in-partially-understood-domains-speaker-adele-h-ribeiro-columbia-university"> Interinstitutional Graduate Program in Statistics (PIPGES)</a> <br>
              Federal University of São Carlos and  University of São Paulo <br>
              <i> <strong>Ribeiro, A. H.</strong> Causal Effect Identification in Partially Understood Domains.</i>
              <a href="https://youtu.be/Hmk0OTtgzms?feature=shared" download target="_blank"> (Talk on Youtube) </a>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Mar 2022</td>
            <td>  <a target="_blank" href="https://www.voices-of-data-science.com/"> Voices of Data Science at UMass Amherst</a> <br>
              Manning College of Information & Computer Sciences, University of Massachusetts Amherst<br>
              <i> <strong>Ribeiro, A. H.</strong>. On the Importance of Causal Inference in the Next Generation of Artificial Intelligence.</i>
              <a href="https://youtu.be/s61549EBmjc?feature=shared" download target="_blank"> (Talk on Youtube) </a>  
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Mar 2022</td>
            <td>  Causal Inference Learning Group (CILG) <br>
              Biostatistics Department, Mailman School of Public Health, Columbia University <br>
              <i> <strong>Ribeiro, A. H.</strong>.Effect Identification in Cluster Causal Diagrams.</a></i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Dec 2021</td>
            <td>  WHY-21 at NeurIPS 2021 - <a target="_blank" href="https://why21.causalai.net/"> Causal Inference &amp; Machine Learning: Why now? </a> <br>
              <i> <strong>Ribeiro, A. H.</strong>. <a href="https://neurips.cc/virtual/2021/workshop/21871" download target="_blank">Effect Identification in Cluster Causal Diagrams.</a></i>
            </td>
          </tr> 
        
            <tr>
            <td style="vertical-align:top"> Nov 2021</td>
            <td> Laboratory of Epidemiology &amp; Population Science (LEPS) at the National Institute on Aging (NIA) <br>
              <i> <strong>Ribeiro, A. H.</strong>. Causal Inference and the Data-Fusion Problem </i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Nov 2021</td>
            <td> <a target="_blank" href="https://www.oecd.org/sti/inno/aiandtheproductivityofscience-speakers.htm"> OECD workshop on AI and the productivity of science. </a> <br>
              <i> <strong>Ribeiro, A. H.</strong>, Bareinboim, E.. Developing causal AI: its importance and an overview. </i>
              <a href="https://www.youtube.com/watch?v=mGbX28TZvw0&t=13954s" download target="_blank"> (Talk on Youtube) </a>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> May 2019 </td>
            <td>  Graduate Seminars Series - Statistics<br>
              Federal University of Sao Carlos and University of Sao Paulo (UFSCar - USP), Sao Carlos, SP, Brazil<br>
              <i> <strong>Ribeiro, A. H.</strong>. Learning Genetic and Environmental Probabilistic Graphical Models from Gaussian Family Data. </i>
            </td>
          </tr> 

        </table>
    </div>
 
     
      </div><!-- /.container -->
   </section><!-- End Talks Section -->



    <!-- Media Section -->
    <section id="media" class="section section-padding">
      
      <div class="container">
      <h2 class="section-title wow fadeInUp">Appearances in Popular Media</h2>

     <div class="list-info wow fadeInUp">
      
      <table class="table-borderless" style="width:100%">
        
        <tr>
            <td style="vertical-align:top"> Oct 2021 </td>
            <td>  <a target="_blank" href="https://www2.daad.de/der-daad/daad-aktuell/en/80920-how-i-would-like-to-continue-my-research-/">“How I would like to continue my research... ”</a><br>
            Interview by Klaus Rathje on the DAAD Postdoctoral Networking Tour "AI in Medicine".
            </td>
        </tr> 
        
        
          <tr>
            <td style="vertical-align:top"> May 2021 </td>
            <td>  <a target="_blank" href="https://datascience.columbia.edu/news/2021/developing-and-applying-causal-inference-methods-in-public-health/">Developing and Applying Causal Inference Methods in Public Health</a><br>
            Interview by Karina Alexanyan, Ph.D., for the Data Science Institute at Columbia University.
            </td>
          </tr> 


        </table>
    </div>
  </div><!-- /.container -->
   </section><!-- End Fellowships Section -->



      <!-- Tutorials/Short-Courses Section -->
      <section id="tutorials" class="section section-padding">
      
        <div class="container">
        <h2 class="section-title wow fadeInUp"> Invited Lectures and Short-Courses</h2>
       
      <div class="list-info wow fadeInUp">
        
        <table class="table-borderless" style="width:100%">

          <tr>
            <td style="vertical-align:top"> Jul 2024 </td>
            <td> <a target="_blank" href="https://essai2024.di.uoa.gr/ESSAI-courses.html"> 2nd European Summer School on Artificial Intelligence - ESSAI 2024 </a>  <br>
              5-day Course <br>
              Department of Informatics and Telecommunications National and Kapodistrian University of Athens, Athens, Greece. <br>
               <strong>Ribeiro, A. H.</strong>, Dhami, D., and Zecevic, M. <i> Machines Climbing Pearl's Ladder of Causation. </i>
               <a href="https://youtube.com/playlist?list=PLIxSri8f3YTWzjobCzBcyRlrE5C81gITH&feature=shared" download target="_blank"> (Lectures on Youtube) </a>  
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Jul 2024 </td>
            <td> <a target="_blank" href="http://lxmls.it.pt/2024/">14th Lisbon Machine Learning School - LxMLS 2024 </a>  <br>
              3-hour Tutorial <br>
              Instituto Superior Técnico, Lisbon, Portugal <br>
              <strong>Ribeiro, A. H.</strong>.<i>  Introduction to Causal Inference. </i> 
              <a href="https://www.youtube.com/live/wx36E8ItZRE?feature=shared" download target="_blank"> (Lecture on Youtube) </a><br>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Jun 2024 </td>
            <td> <a target="_blank" href="https://probabilistic.ai/">Nordic Probabilistic AI School - ProbAI 2024</a>  <br>
              3-hour Tutorial <br>
             Frederiksberg Campus of University of Copenhagen, Copenhagen, Denmark <br>
              <strong>Ribeiro, A. H.</strong>. <i> Introduction to Causal Inference. </i> 
              <a href="https://youtu.be/iXhAXF1133A?feature=shared" download target="_blank"> (Lecture on Youtube) </a> 
            </td>
          </tr> 


          <tr>
            <td style="vertical-align:top"> Jan 2024 </td>
            <td> <a target="_blank" href="https://probabilistic.ai/"> Tropical Probabilistic AI School - Tropical ProbAI 2024</a>  <br>
              3-hour Tutorial <br>
              Hosted with the EMAp FGV Summer School on Data Science 2024, Rio de Janeiro, Brazil <br>
              <a target="_blank" href="https://github.com/adele/Causality-Tutorial"> Tutorial on GitHub. </a><br>
              <strong>Ribeiro, A. H.</strong>. <i> Introduction to Causal Inference. </i>
            </td>
          </tr> 
  
          <tr>
            <td style="vertical-align:top"> Jul 2023 </td>
            <td> <a target="_blank" href="https://essai.si/machines-climbing-pearls-ladder-of-causation/"> European Summer School on Artificial Intelligence - ESSAI 2023 </a>  <br>
              5-day Course <br>
              Faculty of Computer and Information Science, University of Ljubljana, Slovenia <br>
               <strong>Ribeiro, A. H.</strong>, Dhami, D., and Zecevic, M. <i> Machines Climbing Pearl's Ladder of Causation. </i>
            </td>
          </tr> 
  
          <tr>
            <td style="vertical-align:top"> Jul 2023 </td>
            <td> <a target="_blank" href="https://www.youtube.com/watch?v=VA8vzmfnhyk"> 13rd Lisbon Machine Learning School - LxMLS 2023</a>  <br>
              3-hour Tutorial <br>
              Instituto Superior Técnico, Lisbon, Portugal <br>
              <strong>Ribeiro, A. H.</strong>. <i> Causality and its Role in Reasoning, Explainability, and Generalizability.  
                <a href="https://www.youtube.com/live/VA8vzmfnhyk?feature=shared" download target="_blank"> (Lecture on Youtube) </a> </i>
            </td>
          </tr> 
  
          <tr>
            <td style="vertical-align:top"> Jun 2023 </td>
            <td> <a target="_blank" href="https://probabilistic.ai/"> Nordic Probabilistic AI School - ProbAI 2023</a>  <br>
              3-hour Tutorial <br>
              Norwegian University of Science and Technology (NTNU), Trondheim, Norway <br>
              <a target="_blank" href="https://github.com/probabilisticai/probai-2023/tree/main/day_5"> Tutorial on GitHub. </a><br>
              <strong>Ribeiro, A. H.</strong>. <i> Causal Inference: Towards Explainable, Generalizable, and Trustworthy AI. </i>
              <a href="https://youtu.be/NkTtwHEMeEg?feature=shared" download target="_blank"> (Lecture on Youtube) </a>
            </td>
          </tr> 
  
  
          <tr>
            <td style="vertical-align:top"> Feb 2023 </td>
            <td> <a target="_blank" href="https://aaai-23.aaai.org/bridge-program-cfps/"> Continual Causality - Bridge Program at AAAI-2023 </a>  <br>
              90-min Tutorial <br>
              Walter E. Washington Convention Center, Washington DC, USA <br>
              <strong>Ribeiro, A. H.</strong>. <a target="_blank" href="https://www.continualcausality.org/program/"> Putting the Causality in Continual Causality. </a>  </i>
            </td>
          </tr> 
  
            <tr>
              <td style="vertical-align:top"> Jul 2022 </td>
              <td> <a target="_blank" href="http://lxmls.it.pt/2022/index.html#schedule"> 12th Lisbon Machine Learning Summer School (LxMLS - 2022) </a> <br>
                Invited 3-hour Tutorial <br>
                <strong>Ribeiro, A. H.</strong>, Bareinboim, E.. <i> Causal Data Science <a href="https://www.youtube.com/live/r6t8oaWIge4?feature=shared" download target="_blank"> (Lecture on Youtube) </a></i>
              </td>
            </tr> 
  
            <tr>
              <td style="vertical-align:top"> Sep 2021 </td>
              <td>  Graduate Seminars Series - Statistics <br>
                Statistics Department, University of Brasilia - UnB, Brasilia, Brazil<br>
                Invited Lecture <br>
                <strong>Ribeiro, A. H.</strong>.<i><a target="_blank" href="http://www.est.unb.br/index.php?option=com_content&view=article&id=231&catid=9&Itemid=107">  Causal Inference and Data-Fusion. </i></a>
              </td>
            </tr> 
            
            <tr>
              <td style="vertical-align:top"> Jul 2021 </td>
              <td> 11th Lisbon Machine Learning Summer School (LxMLS - 2021)<br>
                Invited 3-hour Tutorial <br>
                <strong>Ribeiro, A. H.</strong>, Bareinboim, E..<i> Causal Data Science: An Introduction to Causal Inference and Data-Fusion. <a href="https://www.youtube.com/live/2zkQjhMiZq4?feature=shared" download target="_blank"> (Lecture on Youtube) </a></i>
              </td>
            </tr> 
            
            <tr>
              <td style="vertical-align:top"> Jun 2021 </td>
              <td>  Perspectives in Statistics <br>
              Statistics Department, University of Sao Paulo (IME - USP), Sao Paulo, SP, Brazil<br>
                Invited Lecture <br>
                <strong>Ribeiro, A. H.</strong>. <i> Causal Inference from Observational Studies </i>
              </td>
            </tr> 
            
            <tr>
              <td style="vertical-align:top"> Dec 2020 </td>
              <td> Seventy-Sixth (76th) Annual Deming Conference on Applied Statistics. <br>
                Invited 3-hour Tutorial <br>
                <strong>Ribeiro, A. H.</strong>,  Adibuzzaman, M., Bareinboim, E.. <i> Causal Inference in the Health Sciences.</i>
              </td>
            </tr> 
          
            <tr>
              <td style="vertical-align:top"> Nov 2020 </td>
              <td> American Medical Informatics Association  (AMIA 2020) Virtual Annual Symposium. <br>
                Contributed 3.5-hour Tutorial <br>
                <strong>Ribeiro, A. H.</strong>,  Adibuzzaman, M., Bareinboim, E.. <i> Causal Inference in the Health Sciences.</i>
              </td>
            </tr>
  
            <tr>
              <td style="vertical-align:top"> Oct 2020 </td>
              <td>  Graduate Seminars Series - Biostatistics and Biometrics <br>
                Sao Paulo State University - UNESP, Botucatu, SP, Brazil<br>
                Invited Lecture <br>
                <strong>Ribeiro, A. H.</strong>. <i> Causal Inference from Observational Studies </i>
              </td>
            </tr>         
  
            <tr>
              <td style="vertical-align:top"> Jan 2017 </td>
              <td> Graduate Summer School - Sao Paulo State University - UNESP, Presidente Prudente, SP, Brazil<br>
                9-hour Short Course <br>
                <strong>Ribeiro, A. H.</strong>,  Soler, J.M.P.. <i> Dimensionality Reduction and Structure Learning with Applications to Genomics</i>
              </td>
            </tr> 
            
            <tr>
              <td style="vertical-align:top"> May 2016 </td>
              <td> 61a Reunião Anual da Região Brasileira da Sociedade Internacional de Biometria (RBras), Salvador, BA, Brazil<br>
                4-hour Short Course <br>
                strong>Ribeiro, A. H.</strong>,  Soler, J.M.P.. <i> Dimensionality Reduction Applied to Genomics</i>
              </td>
            </tr> 
          </table>
      </div>
        </div><!-- /.container -->
     </section><!-- End Tutorial/Short-Courses Section -->
  
  <!-- Teaching Section -->
  
    <section id="teaching" class="section section-padding">
      
    <div class="container">
      <h2 class="section-title wow fadeInUp">Teaching</h2>
      
    <div class="list-info wow fadeInUp">
      <h3>Lecturer </h3>
    
      <table class="table-borderless" style="width:100%">
        <tr>
          <td style="vertical-align:top"> Oct 2023 - Sep 2024 </td>
          <td> Department of Computer Science, Heinrich Heine University of Düsseldorf, Germany<br>
            <i>Courses: Causal Data Science  <a href="https://youtube.com/playlist?list=PLIxSri8f3YTXfOQg1-3IbM_OpqDoahDCt&feature=shared" target="_blank">(Course on Youtube)</a>; Topics in Causality. </i>
          </td>
        </tr>    
      <tr>
        <td style="vertical-align:top"> Mar 2023 - Oct 2023 </td>
        <td> Department of Mathematics and Computer Science, Phillips University of Marburg, Germany<br>
          <i>Course: Causal Data Science: Theoretical Foundations and Algorithms.</i>
        </td>
      </tr>         
    </table>
  </div>  
      
    <div class="list-info wow fadeInUp">
    <h3>Assistant Professor </h3>
    
          <table class="table-borderless" style="width:100%">

          <tr>
            <td style="vertical-align:top"> Feb 2018 - Jul 2018 </td>
            <td> Computer Engineering Department - Institute of Education and Research (Insper), Sao Paulo, SP, Brazil.<br>
              <i>Course: Software Design using Python</i>
            </td>
          </tr>         
        </table>
      </div>  
          
          
    <div class="list-info wow fadeInUp">
    <h3>Teaching Assistant</h3>

      <table class="table-borderless" style="width:100%">
        
          <tr>
            <td style="vertical-align:top"> Mar 2017 - Jul 2017 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Statistical Design of Experiments</i>
            </td>
          </tr>      
          
          <tr>
            <td style="vertical-align:top"> Aug 2016 - Dec 2016 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Multivariate Data Analysis</i>
            </td>
          </tr>         
        
          <tr>
            <td style="vertical-align:top"> Mar 2016 - Jul 2016 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Statistical Methods for Genetics and Genomics</i>
            </td>
          </tr>         
                
          <tr>
            <td style="vertical-align:top"> Aug 2015 - Dec 2015 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Multivariate Data Analysis</i>
            </td>
          </tr>         
        
          <tr>
            <td style="vertical-align:top"> Mar 2015 - Jul 2015 </td>
            <td> Architecture and Urbanism College - University of Sao Paulo (FAU-USP), Sao Paulo, SP, Brazil.<br>
              <i>Mathematics, Architecture and Design</i>
            </td>
          </tr>         
        

          <tr>
            <td style="vertical-align:top"> Aug 2014 - Dec 2014 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Statistical Techniques, Programming and Simulation</i>
            </td>
          </tr>         
        
          <tr>
            <td style="vertical-align:top"> Mar 2014 - Jul 2014 </td>
            <td> Institute of Astronomy, Geophysics and Atmospheric Sciences - University of Sao Paulo (IAG-USP), Sao Paulo, SP, Brazil.<br>
              <i>Numerical Calculus with Applications in Physics</i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Aug 2013 - Dec 2013 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Mathematical Modeling</i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Mar 2013 - Jul 2013 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Introduction to Computer Programming</i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Aug 2012 - Dec 2012 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Linear Programming</i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Mar 2012 - Jul 2012 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Numerical Methods for Linear Algebra</i>
            </td>
          </tr> 
        </table>
    </div>
    
     </div><!-- /.container -->
   </section><!-- End Teaching Section -->


	<!-- Scroll-up -->
	<div class="scroll-up">
		<a href="#about"><i class="fa fa-angle-up"></i></a>
	</div>

	<!-- Javascript files -->
	<script src="assets/js/jquery.js"></script>
	<script src="assets/bootstrap/js/bootstrap.min.js"></script>
  <script src="assets/js/jquery.stellar.min.js"></script>
	<script src="assets/js/jquery.sticky.js"></script>
  <script src="assets/js/smoothscroll.js"></script>
	<script src="assets/js/wow.min.js"></script>
  <script src="assets/js/jquery.countTo.js"></script>
  <script src="assets/js/jquery.inview.min.js"></script> 
  <script src="assets/js/jquery.easypiechart.js"></script>
  <script src="assets/js/jquery.shuffle.min.js"></script>
  <script src="assets/js/jquery.magnific-popup.min.js"></script>
  <script src="http://a.vimeocdn.com/js/froogaloop2.min.js"></script>
  <script src="assets/js/jquery.fitvids.js"></script>
  <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&amp;sensor=false"></script>
  <script src="assets/js/scripts.js"></script>
</body>
</html>
