<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="description" content="Adele Ribeiro's Personal Webpage">
  <meta name="keywords" content="causal inference, resume, data science, AI, healthcare, deep learning, adele, helena, ribeiro">
	<meta name="author" content="Adele H. Ribeiro">

	<title>Adele H. Ribeiro</title>

  <!-- Web Fonts -->
  <link href='http://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
	<!-- Bootstrap core CSS -->
	<link href="assets/bootstrap/css/bootstrap.min.css" rel="stylesheet" media="screen">
	<!-- Font Awesome CSS -->
	<link href="assets/css/font-awesome.min.css" rel="stylesheet" media="screen">
	<!-- Animate css -->
  <link href="assets/css/animate.css" rel="stylesheet">
  <!-- Magnific css -->
	<link href="assets/css/magnific-popup.css" rel="stylesheet">
	<!-- Custom styles CSS -->
	<link href="assets/css/style.css" rel="stylesheet" media="screen">
	<!-- Custom styles CSS -->
	<!-- <link href="assets/css/accordion.css" rel="stylesheet" media="screen">  -->
  <!-- Responsive CSS -->
  <link href="assets/css/responsive.css" rel="stylesheet">
  <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
  <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
  <![endif]-->

  <link rel="shortcut icon" href="">
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="assets/images/ico/apple-touch-icon-144-precomposed.png">
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="assets/images/ico/apple-touch-icon-114-precomposed.png">
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="assets/images/ico/apple-touch-icon-72-precomposed.png">
  <link rel="apple-touch-icon-precomposed" href="assets/images/ico/apple-touch-icon-57-precomposed.png">

</head>

<body>

	<!-- Navigation -->
	<header class="header">
		<nav class="navbar navbar-custom" role="navigation">
		      <div class="container-fluid">
		      
		      
		      <!-- BRAND -->
          <div class="navbar-header">
					<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#custom-collapse">
          
          <span class="sr-only">Toggle navigation</span>
          </button>
          <a class="navbar-brand" href="#"><h1 style="padding-left:15px">Adèle H. Ribeiro</h1></a>
          </div>
		      
		     
				<div class="collapse navbar-collapse" id="custom-collapse">
					<ul class="nav navbar-nav navbar-right">
						<li><a href="#about">About</a></li>
						<li><a href="#resume">Resume</a></li>
						<li><a href="#publications">Publications</a></li>
						<li><a href="#libraries">Libraries</a></li>
						<li><a href="#teaching">Teaching</a></li>
					</ul>
				</div>
		</nav>
		<script>
    window.location = "#about";
</script>
	</header><!-- End Navigation -->
    <!-- About Section -->
    <section id="about" class="about-section section-padding">
      <div class="container">
        <h2 class="section-title wow fadeInUp">About Me</h2>

        <div class="row">

          <div class="col-md-4 col-md-push-8">
            <div class="biography text-center">
                <div class="myphoto">
                  <img src="assets/images/myphoto.jpg" alt="">
                </div>
              
                <p><h4>Adèle Helena Ribeiro</h4>
                <span>Postdoctoral Researcher</span><br>
                <span>Causal AI Lab, Columbia University</span><br>
                <a class="email" href="adele@cs.columbia.edu">adele@cs.columbia.edu</a>

                <div class="social-icons text-center">
                  <ul class="list-inline">
                  
                   <li><a href="https://scholar.google.com/citations?user=Py9MTe4AAAAJ&amp;hl=en">
                    <i class="ai ai-google-scholar"></i>
                  </a></li>
                    
                  <li><a href="http://lattes.cnpq.br/8716519765059982">
                    <i class="ai ai-lattes"></i>
                  </a></li>
                  
                  <li><a href="https://github.com/adele">
                    <i class="fa fa-github"></i>
                  </a></li>
                  
                  
                  <li><a href="https://twitter.com/adelehr">
                    <i class="fa fa-twitter"></i>
                  </a></li>

                  </ul>
                </div> <!-- /.social-icons --> 
             </div>   
          </div> <!-- col-md-4 -->

          <div class="col-md-8 col-md-pull-4">
            <div class="short-info wow fadeInUp">
              <h3>About me</h3>
              <p>I am a Postdoctoral Research Scientist in the Causal Artificial Intelligence (Causal AI) Laboratory, where I work with Professor Elias Bareinboim. My research lies at the intersection of Computer Science, Statistics, and Artificial Intelligence in Healthcare. I am currently interested in the development and application of causal reasoning and decision-making tools in the Health Sciences for discovering, integrating, generalizing, and personalizing cause-effect relationships from heterogeneous observational and experimental data collections. </p>
            </div>

            <div class="short-info wow fadeInUp">
              <h3>Research Interests</h3>
              <p></p>

              <ul class="list-check">
                <li>Causal Inference</li>
                <li>Deep Learning</li>
                <li>Structure Learning</li>
                <li>Statistical Genetics</li>
                <li>AI in Healthcare </li>
                <li>Multi-Omics Analysis</li>
              </ul>
            </div>

           
            <div class="download-button text-center">
              <a class="btn btn-primary btn-lg" href="assets/pdf/RibeiroAH_CV.pdf" download target="_blank"> <i class="fa fa-download" aria-hidden="true"></i> Curriculum vitae</a>
              
            </div>
                            
          </div>


        </div> <!-- /.row -->
      </div> <!-- /.container -->
    </section><!-- End About Section -->


    <!-- Resume Section -->
    <section id="resume" class="resume-section section-padding">
        <div class="container">
            <h2 class="section-title wow fadeInUp">Resume</h2>
            <div class="row">
                <div class="col-md-12">
                    <div class="resume-title">
                        <h3>Education and Professional Preparation</h3>
                    </div>
                    <div class="resume">
                        <ul class="timeline">
                                                         <li>
                                <div class="posted-date">
                                    <span class="month">Sep 2019</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Postdoctoral Scholar<br></h3>
                                            <span> Causal Artificial Intelligence Laboratory<br>
                                            Data Science / Computer Science Institutes <br>
                                            Columbia University<br>
                                            New York, NY, USA </span> 
                                            
                                            <div class="timeline-body"> 
                                            <Strong>Project:</Strong> Causal Inference in the Health Sciences: from Biased and Heterogeneous Data Collections to Personalized and Improved Patient Outcomes. <br>
                                            <Strong>Advisor:</Strong> Prof. Elias Bareinbom
                                             </div><!-- /timeline-body -->
                                             
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            
                            <li class="timeline-inverted">
                                <div class="posted-date">
                                    <span class="month">Jan 2019</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Postdoctoral Scholar</h3>
                                            <span>Laboratory of Genetics and Molecular Cardiology<br>
                                            Heart Institute (InCor) <br>
                                            University of Sao Paulo<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Project:</Strong> Deep Learning for 12-lead ECG Classification.<br>
                                            <Strong>Advisor:</Strong> Prof. José Eduardo Krieger
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                          
                          
                               <li>
                                <div class="posted-date">
                                    <span class="month">Nov 2018</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Doctor of Philosophy in<br>Computer Science</h3>
                                            <span>Institute of Mathematics and Statistics<br>
                                            University of Sao Paulo (IME-USP)<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            
                                            <Strong>PhD's dissertation:</Strong> 
                                            <a class="timeline-link" href="https://doi.org/10.11606/T.45.2019.tde-15032019-190109">Identification of Causality in Genetics and Neuroscience</a><br>
                                            <Strong>Advisor:</Strong> Prof. André Fujita<br>
                                            <Strong>Co-Advisor:</Strong> Prof. Júlia Maria Pavan Soler<br>
                                            
                                             
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            
                            <li class="timeline-inverted">
                                <div class="posted-date">
                                    <span class="month">Fall 2017</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Doctoral Research Internship</h3>
                                            <span>Neuroscience Institute<br>
                                            Princeton University<br>
                                            Princeton, NJ, USA </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Project:</Strong> Deep learning-based pose representation and dynamics modeling of marmoset monkeys.<br>
                                            <Strong>Advisor:</Strong> Prof. Asif A. Ghazanfar
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            <li>
                                <div class="posted-date">
                                    <span class="month">Jun 2014</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Master of Science in
                                            <br>Computer Science</h3>
                                            <span>Institute of Mathematics and Statistics<br>
                                            University of Sao Paulo (IME-USP)<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Master's thesis:</Strong> 
                                            <a class="timeline-link" href="http://dx.doi.org/10.11606/D.45.2014.tde-04082014-163616">Gene expression analysis taking into account measurement errors and application to real data.</a><br>
                                            <Strong>Advisor:</Strong> Prof. Roberto Hirata Jr.<br>
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>
                            
                            <li class="timeline-inverted">
                                <div class="posted-date">
                                    <span class="month">Dec 2011</span>
                                </div><!-- /posted-date -->

                                <div class="timeline-panel wow fadeInUp">
                                    <div class="timeline-content">
                                        <div class="timeline-heading">
                                            <h3>Bachelor of Science in 
                                            Computational<br>and Applied Mathematics</h3>
                                            <span>Institute of Mathematics and Statistics<br>
                                            University of Sao Paulo (IME-USP)<br>
                                            Sao Paulo, SP, Brazil </span> 
                                            <div class="timeline-body"> 
                                            <Strong>Senior thesis:</Strong> Analysis of Pyroelectric Infrared (PIR) sensor output signals.<br>
                                            <Strong>Advisor:</Strong> Prof. Roberto Hirata Jr.
                                             </div><!-- /timeline-body -->
                                             </div><!-- /timeline-heading -->
                                    </div> <!-- /timeline-content -->
                                </div><!-- /timeline-panel -->
                            </li>

                      </ul>
                    </div>
                </div>
            </div><!-- /row -->
        </div><!-- /.container -->
    </section><!-- End Resume Section -->


    
    <!-- Publications Section -->
    <section id="publications" class="section section-padding">
      
      <div class="container">
      <h2 class="section-title wow fadeInUp">Publications</h2>
    
        <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
        
        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="heading5">Anand, T., <Strong>Ribeiro, A. H.</Strong>, Tian, J. , Bareinboim, E. (2021). Effect Identification in Causal Diagrams with Clustered Variables.<i> Columbia CausalAI Laboratory, Technical Report (R-77). </i> <a href="https://causalai.net/r77.pdf">https://causalai.net/r77.pdf</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" data-target="#collapse5" aria-expanded="false" aria-controls="collapse5"></a></td>
          </tr> 
        </table>
        <div id="collapse5" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading5">
            <div class="panel-body">
              One pervasive task found throughout the empirical sciences is to determine the effect of interventions  from non-experimental data. It is well-understood that assumptions are necessary to perform such causal inferences, an idea popularized through Cartwright's motto: "no causes-in, no causes-out." One   way of articulating these assumptions is through the use of causal diagrams, which are a special type of graphical model with causal semantics [Pearl, 2000]. The graphical approach has been applied successfully in many settings, but there are still challenges to its use,  particularly in complex,  high-dimensional domains. In medicine, for example,  background knowledge may exist about the relationships among a subset of variables, but usually not all of them. In this paper, we introduce <i>cluster causal diagrams</i> (for short, C-DAGs) to allow for the representation of partial understanding of the relationships among variables, which has the potential of alleviating  DAGs' somewhat stringent requirements. C-DAGs provide a simple yet effective way to partially abstract a grouping of variables among which causal relationships are not fully understood. Our goal is to develop machinery to reason on top of C-DAG's new representation. In particular, we first define a new version of the d-separation criterion, and prove its soundness and completeness. Second, we extend these new separation rules and prove the validity of the corresponding do-calculus. Lastly, we show that a standard identification algorithm can systematically compute causal effects from observational data with cluster causal diagrams.
            </div>
          </div>
       </div>
       
       <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="heading4"><Strong>Ribeiro, A. H.</Strong>, Soler, J. M. P. (2020). Learning Genetic and Environmental Graphical Models from Gaussian Family Data.<i> Statistics in Medicine.</i> 39: 2403– 2422. <a href="https://doi.org/10.1002/sim.8545">https://doi.org/10.1002/sim.8545</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" data-target="#collapse4" aria-expanded="false" aria-controls="collapse4"></a></td>
          </tr> 
        </table>
        <div id="collapse4" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading4">
            <div class="panel-body">
              Many challenging problems in biomedical research rely on understanding how variables are associated with each other and influenced by genetic and environmental factors. Probabilistic graphical models (PGMs) are widely acknowledged as a very natural and formal language to describe relationships among variables and have been extensively used for studying complex diseases and traits. In this work, we propose methods that leverage observational Gaussian family data for learning a decomposition of undirected and directed acyclic PGMs according to the influence of genetic and environmental factors. Many structure learning algorithms are strongly based on a conditional independence test. For independent measurements of normally distributed variables, conditional independence can be tested through standard tests for zero partial correlation. In family data, the assumption of independent measurements does not hold since related individuals are correlated due to mainly genetic factors. Based on univariate polygenic linear mixed models, we propose tests that account for the familial dependence structure and allow us to assess the significance of the partial correlation due to genetic (between-family) factors and due to other factors, denoted here as environmental (within-family) factors, separately. Then, we extend standard structure learning algorithms, including the IC/PC and the really fast causal inference (RFCI) algorithms, to Gaussian family data. The algorithms learn the most likely PGM and its decomposition into two components, one explained by genetic factors and the other by environmental factors. The proposed methods are evaluated by simulation studies and applied to the Genetic Analysis Workshop 13 simulated dataset, which captures significant features of the Framingham Heart Study.
            </div>
          </div>
       </div>
       
        <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="heading3"><Strong>Ribeiro, A. H.</Strong>, Soler, J. M. P., Hirata Jr., R. (2019). Variance-Preserving Estimation of Intensity Values Obtained from Omics Experiments.<i>  Frontiers in Genetics.</i> 10:855. <a href="https://www.frontiersin.org/article/10.3389/fgene.2019.00855">https://doi.org/110.3389/fgene.2019.00855</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" data-target="#collapse3" aria-expanded="false" aria-controls="collapse3"></a></td>
          </tr> 
        </table>
        <div id="collapse3" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading3">
            <div class="panel-body">
              Faced with the lack of reliability and reproducibility in omics studies, more careful and robust methods are needed to overcome the existing challenges in the multi-omics analysis. In conventional omics data analysis, signal intensity values (denoted by M and values) are estimated neglecting pixel-level uncertainties, which may reflect noise and systematic artifacts. For example, intensity values from two-color microarray data are estimated by taking the mean or median of the pixel intensities within the spot and then subjected to a within-slide normalization by LOWESS. Thus, focusing on estimation and normalization of gene expression profiles, we propose a spot quantification method that takes into account pixel-level variability. Also, to preserve relevant variation that may be removed in LOWESS normalization with poorly chosen parameters, we propose a parameter selection method that is parsimonious and considers intrinsic characteristics of microarray data, such as heteroskedasticity. The usefulness of the proposed methods is illustrated by an application to real intestinal metaplasia data. Compared with the conventional approaches, the analysis is more robust and conservative, identifying fewer but more reliable differentially expressed genes. Also, the variability preservation allowed the identification of new differentially expressed genes. Using the proposed approach, we have identified differentially expressed genes involved in pathways in cancer and confirmed some molecular markers already reported in the literature.
            </div>
          </div>
       </div>
       
      <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="heading2"><Strong>Ribeiro, A. H.</Strong>, Lotufo, P., Fujita, A, Goulart, A., Chor, D., Mill, J. G., Bensenor, I., Santos, I. S. (2017). Association Between Short-Term Systolic Blood Pressure Variability and Carotid Intima-Media Thickness in ELSA-Brasil Baseline.<i> American Journal of Hypertension.</i> 30:954–960. <a href="http://dx.doi.org/10.1093/ajh/hpx076">http://dx.doi.org/10.1093/ajh/hpx076</a> </td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" data-target="#collapse2" aria-expanded="false" aria-controls="collapse2"></a></td>
          </tr> 
        </table>
        <div id="collapse2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading2">
            <div class="panel-body">
              Blood pressure (BP) is associated with carotid intima-media thickness (CIMT), but few studies have explored the association between BP variability and CIMT. We aimed to investigate this association in the Brazilian Longitudinal Study of Adult Health (ELSA-Brasil) baseline. We found a small but significant association between SBP variability and CIMT values. This was additive to the association between SBP central tendency and CIMT values, supporting a role for high short-term SBP variability in atherosclerosis.
            </div>
          </div>
       </div>
       
               <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="heading1"><Strong>Ribeiro, A. H.</Strong>,  Soler, J. M. P., Neto, E. C., Fujita, A. (2016). Causal Inference and Structure Learning of Genotype-Phenotype Networks Using Genetic Variation.<i> In Big Data Analytics in Genomics. </i> Springer International Publishing, New York, p. 89-143. <a href="http://dx.doi.org/10.1007/978-3-319-41279-5_3">http://dx.doi.org/10.1007/978-3-319-41279-5_3</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" data-target="#collapse1" aria-expanded="false" aria-controls="collapse1"></a></td>
          </tr> 
        </table>
        <div id="collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading1">
            <div class="panel-body">
              A major challenge in biomedical research is to identify causal relationships among genotypes, phenotypes, and clinical outcomes from high-dimensional measurements. Causal networks have been widely used in systems genetics for modeling gene regulatory systems and for identifying causes and risk factors of diseases. In this chapter, we describe fundamental concepts and algorithms for constructing causal networks from observational data. In biological context, causal inferences can be drawn from the natural experimental setting provided by Mendelian randomization, a term that refers to the random assignment of genotypes at meiosis. We show that genetic variants may serve as instrumental variables, improving estimation accuracy of the causal effects. In addition, identifiability issues that commonly arise when learning network structures may be overcome by using prior information on genotype–phenotype relations.
            </div>
          </div>
       </div>
       
     </div>
    </div><!-- /.container -->
   </section><!-- End Publications Section -->

 
 
    <!-- Libraries Section -->
    <section id="libraries" class="section section-padding">
      
      <div class="container">
      <h2 class="section-title wow fadeInUp">Open-Source Libraries</h2>
    
        <div class="panel-group" id="accordion-libraries" role="tablist" aria-multiselectable="true">
       
      <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="heading2-libraries"><Strong>Ribeiro, A. H.</Strong> (2019). FamilyBasedPGMs: an R package for learning genetic and environmental graphical models from family data. <a href="https://github.com/adele/FamilyBasedPGMs">https://github.com/adele/FamilyBasedPGMs</a> </td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion-libraries" data-target="#collapse2-libraries" aria-expanded="false" aria-controls="collapse2-libraries"></a></td>
          </tr> 
        </table>
        <div id="collapse2-libraries" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading2-libraries">
            <div class="panel-body">
              This package provides methods for learning, from observational Gaussian family data (i.e., Gaussian data clusterized in families), Gaussian undirected and directed acyclic PGMs describing linear relationships among multiple phenotypes and a decomposition of the learned PGM into unconfounded genetic and environmental PGMs. Methods are based on zero partial correlation tests derived in the work by <a href="#heading4">Ribeiro and Soler (2020)</a>.
            </div>
          </div>
       </div>
       
               <div class="panel panel-default">
        <table style="width:100%">
          <tr>
            <td class="panel-heading" role="tab" id="heading1-libraries"><Strong>Ribeiro, A. H.</Strong> (2019). OmicsMA: An R Package for Variance-Preserving Estimation and Normalization of M-A Values from Omics Experiments. <a href="https://github.com/adele/omicsMA">https://github.com/adele/omicsMA</a></td>
            <td><a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" data-target="#collapse1-libraries" aria-expanded="false" aria-controls="collapse1-libraries"></a></td>
          </tr> 
        </table>
        <div id="collapse1-libraries" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading1-libraries">
            <div class="panel-body">
              This package provides methods for estimating and normalizing the M (intensity log-ratio) and A (mean log intensity) values from two-channel (or two-color) microarrays. Unlike conventional estimation methods which take into account only measures of location (e.g., mean and median) of the pixel intensities of each channel, the provided estimation method takes into account pixel-level variability, which may reflects uncertainties due noise and systematic artifacts.  
            </div>
          </div>
       </div>
       
     </div>
    </div><!-- /.container -->
   </section><!-- End Publications Section -->

  
  <!-- Teaching Section -->
  
    <section id="teaching" class="section section-padding">
      
    <div class="container">
      <h2 class="section-title wow fadeInUp">Teaching</h2>
      
     
    <div class="list-info wow fadeInUp">
      <h3>Invited Lectures, Short Courses, and Tutorials</h3>
 
      <table class="table-borderless" style="width:100%">
        
          <tr>
            <td style="vertical-align:top"> Jul 2021 </td>
            <td> 11th Lisbon Machine Learning Summer School (LxMLS - 2021)<br>
              Invited 3-hour Tutorial <br>
              <i> <strong>Ribeiro, A. H.</strong>, Bareinboim, E.. <a href="https://youtu.be/2zkQjhMiZq4?list=PLQl_xdhSmQegzsLin54NbfePFAuTUEmUj&t=31" download target="_blank">Causal Data Science: An Introduction to Causal Inference and Data-Fusion.</a></i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Jun 2021 </td>
            <td>  Perspectives in Statistics <br>
            Statistics Department, University of Sao Paulo (IME - USP), Sao Paulo, SP, Brazil<br>
              Invited Lecture <br>
              <i> <strong>Ribeiro, A. H.</strong>. Causal Inference from Observational Studies </i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Dec 2020 </td>
            <td> Seventy-Sixth (76th) Annual Deming Conference on Applied Statistics. <br>
              Invited 3-hour Tutorial <br>
              <i>  <strong>Ribeiro, A. H.</strong>,  Adibuzzaman, M., Bareinboim, E.. Causal Inference in the Health Sciences.</i>
            </td>
          </tr> 
        
          <tr>
            <td style="vertical-align:top"> Nov 2020 </td>
            <td> American Medical Informatics Association  (AMIA 2020) Virtual Annual Symposium. <br>
              Contributed 3.5-hour Tutorial <br>
              <i> <strong>Ribeiro, A. H.</strong>,  Adibuzzaman, M., Bareinboim, E.. Causal Inference in the Health Sciences.</i>
            </td>
          </tr>

          <tr>
            <td style="vertical-align:top"> Oct 2020 </td>
            <td>  Graduate Seminars Series - Biostatistics and Biometrics <br>
              Sao Paulo State University - UNESP, Botucatu, SP, Brazil<br>
              Invited Lecture <br>
              <i> <strong>Ribeiro, A. H.</strong>. Causal Inference from Observational Studies </i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> May 2019 </td>
            <td>  Graduate Seminars Series - Statistics<br>
              Federal University of Sao Carlos and University of Sao Paulo (UFSCar - USP), Sao Carlos, SP, Brazil<br>
              Invited Lecture <br>
              <i> <strong>Ribeiro, A. H.</strong>. Learning Genetic and Environmental Probabilistic Graphical Models from Gaussian Family Data. </i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Jan 2017 </td>
            <td> Graduate Summer School - Sao Paulo State University - UNESP, Presidente Prudente, SP, Brazil<br>
              9-hour Short Course <br>
              <i> <strong>Ribeiro, A. H.</strong>,  Soler, J.M.P.. Dimensionality Reduction and Structure Learning with Applications to Genomics</i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> May 2016 </td>
            <td> 61a Reunião Anual da Região Brasileira da Sociedade Internacional de Biometria (RBras), Salvador, BA, Brazil<br>
              4-hour Short Course <br>
              <i><strong>Ribeiro, A. H.</strong>,  Soler, J.M.P.. Dimensionality Reduction Applied to Genomics</i>
            </td>
          </tr> 
        </table>

    </div>
    
    <div class="list-info wow fadeInUp">
    <h3>Assistant Professor </h3>
    
          <table class="table-borderless" style="width:100%">

          <tr>
            <td style="vertical-align:top"> Feb 2018 - Jul 2018 </td>
            <td> Computer Engineering Department - Institute of Education and Research (Insper), Sao Paulo, SP, Brazil.<br>
              <i>Software Design using Python</i>
            </td>
          </tr>         
        </table>
      </div>  
          
          
    <div class="list-info wow fadeInUp">
    <h3>Teaching Assistant</h3>

      <table class="table-borderless" style="width:100%">
        
          <tr>
            <td style="vertical-align:top"> Mar 2017 - Jul 2017 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Statistical Design of Experiments</i>
            </td>
          </tr>      
          
          <tr>
            <td style="vertical-align:top"> Aug 2016 - Dec 2016 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Multivariate Data Analysis</i>
            </td>
          </tr>         
        
          <tr>
            <td style="vertical-align:top"> Mar 2016 - Jul 2016 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Statistical Methods for Genetics and Genomics</i>
            </td>
          </tr>         
                
          <tr>
            <td style="vertical-align:top"> Aug 2015 - Dec 2015 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Multivariate Data Analysis</i>
            </td>
          </tr>         
        
          <tr>
            <td style="vertical-align:top"> Mar 2015 - Jul 2015 </td>
            <td> Architecture and Urbanism College - University of Sao Paulo (FAU-USP), Sao Paulo, SP, Brazil.<br>
              <i>Mathematics, Architecture and Design</i>
            </td>
          </tr>         
        

          <tr>
            <td style="vertical-align:top"> Aug 2014 - Dec 2014 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Statistical Techniques, Programming and Simulation</i>
            </td>
          </tr>         
        
          <tr>
            <td style="vertical-align:top"> Mar 2014 - Jul 2014 </td>
            <td> Institute of Astronomy, Geophysics and Atmospheric Sciences - University of Sao Paulo (IAG-USP), Sao Paulo, SP, Brazil.<br>
              <i>Numerical Calculus with Applications in Physics</i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Aug 2013 - Dec 2013 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Mathematical Modeling</i>
            </td>
          </tr> 

          <tr>
            <td style="vertical-align:top"> Mar 2013 - Jul 2013 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Introduction to Computer Programming</i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Aug 2012 - Dec 2012 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Linear Programming</i>
            </td>
          </tr> 
          
          <tr>
            <td style="vertical-align:top"> Mar 2012 - Jul 2012 </td>
            <td> Institute of Mathematics and Statistics - University of Sao Paulo (IME-USP), Sao Paulo, SP, Brazil.<br>
              <i>Numerical Methods for Linear Algebra</i>
            </td>
          </tr> 
        </table>
    </div>
    
     </div><!-- /.container -->
   </section><!-- End Publications Section -->


	<!-- Scroll-up -->
	<div class="scroll-up">
		<a href="#home"><i class="fa fa-angle-up"></i></a>
	</div>

	<!-- Javascript files -->
	<script src="assets/js/jquery.js"></script>
	<script src="assets/bootstrap/js/bootstrap.min.js"></script>
  <script src="assets/js/jquery.stellar.min.js"></script>
	<script src="assets/js/jquery.sticky.js"></script>
  <script src="assets/js/smoothscroll.js"></script>
	<script src="assets/js/wow.min.js"></script>
  <script src="assets/js/jquery.countTo.js"></script>
  <script src="assets/js/jquery.inview.min.js"></script> 
  <script src="assets/js/jquery.easypiechart.js"></script>
  <script src="assets/js/jquery.shuffle.min.js"></script>
  <script src="assets/js/jquery.magnific-popup.min.js"></script>
  <script src="http://a.vimeocdn.com/js/froogaloop2.min.js"></script>
  <script src="assets/js/jquery.fitvids.js"></script>
  <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&amp;sensor=false"></script>
  <script src="assets/js/scripts.js"></script>
</body>
</html>
